spec/requests/users_controller_spec.rb

```diff
@@ -1783,18 +1783,18 @@
           end
 
           it "shows flash notice" do
-            patch("/users/#{other_user.id}", headers:, params:)
+            request
 
-            expect(flash[:notice]).to eq("An email has been sent to #{new_email} to confirm this change.")
+            expect(flash[:notice]).to eq("Your account details have been updated.")
```

Comment:
If this is a data provider changing their own email, would they still see something that indicated there'd be a confirmation email, or is that just for support?

Reply:
All users (including providers) looking at their own accounts should see a version with blue banner and the bit that says a confirmation email has been sent if they have a pending email change